### PR TITLE
Tax API adjustments for dashboard

### PR DESCRIPTION
**Pull request by maarcingebala**
Fixes https://github.com/saleor/saleor-dashboard/issues/2407

Two adjustments for flat rates API:
- ignore input items with `null` rates in `taxCountryConfigurationUpdate` mutation
- allow deleting individual tax rates for specific countries in `taxClassUpdate` mutation

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6391 Simple taxes](https://app.clickup.com/t/2549495/SALEOR-6391)

File: saleor/graphql/tax/tests/mutations/test_tax_class_update.py [hunk 1]
```diff
@@ -141,3 +141,33 @@ def test_remove_country_rates(staff_api_client, permission_manage_taxes):
     data = content["data"]["taxClassUpdate"]
     assert not data["errors"]
     assert len(data["taxClass"]["countries"]) == 0
+
+
+def test_remove_individual_country_rates(staff_api_client, permission_manage_taxes):
```
Comment (fowczarek) [inline saleor/graphql/tax/tests/mutations/test_tax_class_update.py:146]: Could you add a test about removing not existing rate?

**Review by fowczarek — APPROVED** [review]

File: saleor/graphql/tax/tests/mutations/test_tax_class_update.py (hunk 1 above)
Comment (maarcingebala) [reply saleor/graphql/tax/tests/mutations/test_tax_class_update.py:146]: Added

File: saleor/graphql/tax/mutations/tax_class_update.py [hunk 2]
```diff
@@ -26,14 +38,17 @@ class TaxClassUpdateError(Error):
...
+        CountryRateUpdateInput,
         description=(
             "List of country-specific tax rates to create or update for this tax class."
         ),
     )
     remove_country_rates = NonNullList(
         CountryCodeEnum,
-        description="List of country codes for which to remove the tax class rates.",
+        description=(
+            "List of country codes for which to remove the tax class rates. Note: it "
```
Comment (kadewu) [inline saleor/graphql/tax/mutations/tax_class_update.py:49]:
```suggestion
            "List of country codes for which to remove the tax class rates. Note: It "
```

Just capitalization after colon

**Review by kadewu — APPROVED** [review]

**Pull request merged**